tests/target_configs.yaml

```diff
@@ -1,5 +1,30 @@
+AndroidTarget:
```

Comment:
Sorry, I didn't mean it would help in automating android emulators.

I was thinking that for a user who just wants to check out devlib and run the tests to validate some changes they made, it could be useful to provide a "default" config that just works out of the box with no additional setup required. (I'm thinking going forward if we have a greater set of tests to run). 

The "example" version would then provide details of all the available options including the android emulators that requrie manually starting to provide a more comprehensive testing setup.